### ISSUE TITLE
[Help] How can I dynamically add or remove functions at runtime?

### ISSUE THREAD

**Issue by yanxiyue** [inngest/inngest#540]
I want to implement a workflow platform based on inngest scheduling, on which users can arrange their own business logic, and each workflow is mapped to an inngest function。

Is there any problem with using Inngest in this scenario? if so, how can I  dynamically add or remove functions at runtime?

**Comment by mdmjg** [comment]
Hi, can we get an update on this?

**Comment by lightyaer** [comment]
Yeah, I'm in a similar problem, I was using mergent.co for this, which worked flawlessly, but since resend acquired it and it's sunsetting now, there is no good alternative for this.
